### ISSUE TITLE
I reproduced pix2pix training on apple silicon

### ISSUE THREAD

**Issue by andjoer** [GaParmar/img2img-turbo#29]
Hello, 

thanks for your awesome work! I modified the train pix2pix code to work on apple silicon (I have a branch "apple-silicon" on my fork on GitHub). It was straight forward but there is one incompatibility with vision-aided-loss (line 128 of cvmodel.py) 
`x = F.interpolate(x*0.5+0.5, size=(224, 224), mode='area')`
Since the mode is area on mps devices the input size needs to be divisible by 224, so I added 
`x_tgt_resized = F.interpolate(x_tgt, size=(448, 448), mode='bilinear')
  x_tgt_pred_resized = F.interpolate(x_tgt, size=(448, 448), mode='bilinear')`

before entering the x_tgt and x_tgt_pred into the discriminator. 

Question:
It trains well (takes around 15h on a MacBook Pro M3) but  I wonder if and how much this affects the training characteristics. Could anyone have a wandb report of the my_fill50k dataset training  from a run with the standard code on Cuda? 

To reproduce: 
After installing vision-aided-loss you have to uninstall PyTorch and install the nightly version again. Otherwise it does not work on apple mps anymore.
